Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/ruby-26
+FROM ruby:alpine as builder
 ENV HEIMDALLTOOLS_VERSION 1.3.6
 MAINTAINER rx294@nyu.edu
```

Comment:
I'd probably go ahead and just remove the MAINTAINER tag on this since you just reworked the whole file.